tutorials/introduction/training_config/UT1.jsonnet

```diff
@@ -0,0 +1,93 @@
+// Configuration for the CONLL model from AllenAI, modified slightly
```

Comment:
Should probably move these 3 files to test/integration_tests since they have nothing to do with the tutorial now?